### PR TITLE
Add Jenkinsfile

### PR DESCRIPTION
**Pull request by smillidge**
Signed-off-by: Steve Millidge <steve.millidge@payara.fish>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+env.label = "orb-ci-pod-${UUID.randomUUID().toString()}"
+pipeline {
+  options {
+    // keep at most 50 builds
+    buildDiscarder(logRotator(numToKeepStr: '50'))
+    // abort pipeline if previous stage is unstable
+    skipStagesAfterUnstable()
+    // show timestamps in logs
+    timestamps()
+    // global timeout, abort after 6 hours
```
Comment (pzygielo) [inline Jenkinsfile:26]:
```suggestion
    // global timeout
```

**Comment by smillidge** [comment]
@pzygielo I have a working Jenkinsfile now

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
```
Comment (pzygielo) [inline Jenkinsfile:2]:
```suggestion
 * Copyright (c) 2020 Contributors to the Eclipse Foundation.
```
perhaps?

**Comment by pzygielo** [comment]
Cosmetic - but could indentation be used consistently (e.g. 2 spaces) and fixed in tools & steps, please?

**Comment by smillidge** [comment]
@edbratt @m0mus I need a review on this

**Review by m0mus — APPROVED** [review]
LGTM

**Pull request merged**